cmd/snap-preseed/preseed_linux.go

```diff
@@ -235,10 +235,11 @@ func prepareChroot(preseedChroot string) (*targetSnapdInfo, func(), error) {
 	}
 
 	fstype, fsopts := squashfs.FsType()
-	cmd := exec.Command("mount", "-t", fstype, "-o", strings.Join(fsopts, ","), coreSnapPath, where)
-	if err := cmd.Run(); err != nil {
+	mountArgs := []string{"-t", fstype, "-o", strings.Join(fsopts, ","), coreSnapPath, where}
+	cmd := exec.Command("mount", mountArgs...)
+	if out, err := cmd.CombinedOutput(); err != nil {
 		removeMountpoint()
-		return nil, nil, fmt.Errorf("cannot mount %s at %s in preseed mode: %v ", coreSnapPath, where, err)
+		return nil, nil, fmt.Errorf("cannot mount %s at %s in preseed mode: %v\n'mount %s' failed with: %s", coreSnapPath, where, err, strings.Join(mountArgs, " "), out)
```

Comment:
There's osutil.OutputErr() which can handle mulitple lines in the output, so maybe something like:
```
fmt.Errorf("cannot mount %s at %s in preseed mode, 'mount %s' failed with: %v",
    coreSnapPath, where, strings.Join(mountArgs, " "), osutil.OutputErr(out, err))
```

Reply:
Indeed, unfortunately it ignores original error if output is not empty, so the part that says "exit status 32" is missing.